schema-guide.md

```diff
@@ -748,36 +941,507 @@ Usage example:
 
 `definitions` and its subkeys should not be used directly in `CITATION.cff` files.
 
-- type: ...
-- required: false
+- type: (array of) [`definitions.license-enum`](#definitions.license-enum) objects.
+- required: `false`
 
-...
+License identifier(s) under which the work is made available. When there are multiple licenses, it is assumed their relationship is OR, not AND.
 
-Usage example:
+Usage examples:
 
 ```yaml
+license: Apache-2.0
+```
+```yaml
+license:
+  - Apache-2.0
+  - MIT
 ```
 
 ### `definitions.license-enum`
 
 `definitions` and its subkeys should not be used directly in `CITATION.cff` files.
 
-- type: ...
-- required: false
+- type: `enum` with values:
+    - `0BSD`
```

Comment:
No way around this big list I guess. We could contract into a comma-separated list to save screen space, but this would probably make it less usable, what do you think?

Reply:
yeah there isnt really a good way of dealing with this. Maybe a table could work, but I think I'm right in saying you cant have a table nested in a list. Alternatively, we could do a fixed-width font multicolumn print, like what you would see in a terminal, to help people not injure themselves scrolling through the license list.